contrib/helm/pganalyze-collector/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v2
+name: pganalyze-collector
+version: 0.1.0
```

Comment:
With helm, the chart version and app version are two different versions (they don't strictly have to be the same).

I used an app version of `stable` below as that is what the docs recommended, but it would technically be better to use the exact app version.

The app and chart version can certainly match, but it does present issues if the chart is updated without a corresponding app update.

Reply:
I've updated `appVersion` to `v0.43.1`, but left `version` as is for now. Let me know if you want to keep `appVersion` and `version` in sync. That does make version management easy, but also makes it quite difficult to update the helm chart independently of an app upgrade.

Reply:
Thanks!

To understand correctly, can we keep `version` the same when `appVersion` changes, or do we also need to bump `version` in that case? (e.g. if 0.44.0 is released, would we need to bump `version` to something like `0.2.0`?)

If thats the case (i.e. bumping `version` is needed), I would prefer if we match `version` to be `appVersion`. If `version` doesn't need to be modified when a new `appVersion` is set, then this approach here makes sense.

Reply:
@lfittl Ideally, you should update `version` when `appVersion` changes. Basically, you should bump `version` every time you want to release a new version of the chart (new `appVersion`, new variables, etc). Matching certainly makes things easy on the consume as you only need to worry about the app version, but it does mean that you'll likely only release chart changes with new app versions.

Another thing to consider is that [`version` is supposed to be a strict SemVer version](https://helm.sh/docs/topics/charts/#charts-and-versioning).

I've gone ahead ahead and made them match for now.

Reply:
@jalaziz FYI, I think you ended up committing something different by accident in https://github.com/pganalyze/collector/pull/273/commits/685bca18972de86b7e1e0b2cb07bb6ede49cf177 - I can make the change to match them when merging (and thanks for explaining the difference between the two)

Reply:
> @jalaziz FYI, I think you ended up committing something different by accident in https://github.com/pganalyze/collector/commit/685bca18972de86b7e1e0b2cb07bb6ede49cf177 - I can make the change to match them when merging (and thanks for explaining the difference between the two)

Oops, was in the wrong directory when committing and pushed a test values.yaml. My bad. I need more sleep 😂

Reply:
I just forced pushed to remove the bad `values.yaml` from my branch history but kept your changes. Nothing too sensitive there and it's already public anyway, but figured it's best to remove to avoid confusion.